dojo/settings/settings.dist.py

```diff
@@ -1456,6 +1457,7 @@ def saml2_attrib_map_format(dict):
     'Whispers': DEDUPE_ALGO_HASH_CODE,
     'Blackduck Hub Scan': DEDUPE_ALGO_HASH_CODE,
     'BlackDuck API': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL,
+    'BlackDuck Binary Analysis': DEUPE_ALGO_HASH_CODE,
```

Comment:
Typo here. DEUPE_ALGO_HASH_CODE vs. DEDUPE_ALGO_HASH_CODE

Reply:
Maybe DEDUPE_ALGO_UNITQUE_ID_FROM_TOOL coulld be better with Object SHA1 ?

Reply:
Unfortunately, no.  The object sha1 is a sha1 hash of the object binary itself which resides within a given component (e.g. think of a binary inside of an RPM).  A common use case is you may have different components (i.e. RPMs) containing the same object binary  / object sha1 hash.   Given this common use case, the likelihood of accidental dedupe would be high since patching these components may have different remediation efforts and need to be tracked separately.